src/app/admin/_lib/validationSchemas/createOrganizationSchema.js

```diff
@@ -0,0 +1,109 @@
+import { z } from 'zod';
+import { FormatOfWork } from '@prisma/client';
+import { minMaxString } from '@/lib/validationSchemas/utils';
+import { PHONE_REGEX } from '@/lib/consts';
+
+const DefaultSchema = z.object({
+  name: minMaxString(1, 128, 'Назва'),
+});
+
+const TypeSchema = z.array(minMaxString(1, 64, 'Тип організації'), {
+  required_error: "Тип організації - обов'язкове поле",
+});
+
+const TherapiesSchema = z.array(minMaxString(1, 64, 'Тип терапії'), {
+  required_error: "Тип терапії - обов'язкове поле",
+});
+
+const FormatOfWorkSchema = z
```

Comment:
🟢 You could use Zod enums instead of refining. https://zod.dev/?id=zod-enums

Reply:
We use prisma-generated enums here for syncing with db

Reply:
changed it